### ISSUE TITLE
(Oneechan) Image thumbnail gets cut off when hovering over a quotelink

### ISSUE THREAD

**Issue by revumber** [TuxedoTako/4chan-xt#44]
My setup as follows:
Chromium 122.0.6261.111
uBlock Origin 1.57.0
Violent Monkey 2.18.0
4chan XT 2.7.0
OneeChan 5.9.3
Quote previewing is enabled in 4chan XT.

Browser console log:
![image](https://github.com/TuxedoTako/4chan-xt/assets/134345229/144974bb-cd17-49d5-bf2f-be5586cca612)

(Oneechan is enabled) Hovering over a quotelink shows a truncated thumbnail when previewed reply has a thumbnail close to 1:1 ratio:
![image](https://github.com/TuxedoTako/4chan-xt/assets/134345229/6597b78d-8e55-467d-99c1-96216a36ca3e)

(Oneechan is disabled) The same thumbnail is now unaffected:
![image](https://github.com/TuxedoTako/4chan-xt/assets/134345229/2bf9cc3d-c7cb-4680-a08e-725ed4c67366)

Hover previews for OP and different aspect ratio thumbnail don't have the same problem:
![image](https://github.com/TuxedoTako/4chan-xt/assets/134345229/5877bf63-8d11-4ba3-baf9-10d0e4cb4616)

![image](https://github.com/TuxedoTako/4chan-xt/assets/134345229/dfd20d9c-5152-4a8f-9d89-6c84d46df766)

Don't really have any other information since I just switched from 4chan x to 4chan XT.

**Comment by saxamaphone69** [comment]
I'd argue it's OneeChan at fault rather than 4chan XT, and maybe something in someone's CSS due to some differences between the two userscripts. As you showed, it works fine without OneeChan enabled.

Could you export your OneeChan theme? It's interesting that it only happens on near 1:1 thumbnails. Alternatively, when you inspect the thumbnail's element like you did in the first picture, have a look at which CSS is being applied and if you toggle it on/off does it fix it.

**Comment by revumber** [comment]
OneeChan settings + theme export: https://gist.github.com/revumber/f9fc092c1298e58080fac5a448f78152

Tried inspecting the styles on a live post, but I didn't find anything by toggling the active styles on and off.

**Comment by TuxedoTako** [comment]
I found the problem, because the image thumb is a `display: float`, the actual post is smaller than the hover
![msedge_vftXtZxy5J](https://github.com/TuxedoTako/4chan-xt/assets/126348645/784f2e52-3263-44a3-b0c7-3f4404617af0)
I added that `clip` for those special utf-8 characters that extend to outside the posts.
With custom CSS `#qp .post { overflow: auto; }` the image is fully shown. I'm not good enough at CSS to understand why. [According to mdn `auto` clips too.](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#auto) Why is that one smart enough to extend but `clip` isn't?
